README.md

```diff
@@ -78,6 +81,101 @@ module "apiary-waggledance" {
 }
 ```
 
+### local_metastores
+
+A list of maps.  Each map entry describes a federated metastore server directly accessible on the local network.
+
+An example entry looks like:
+```
+local_metastores = [
+    {
+      host                  = "hms-readonly.metastore.svc.cluster.local"
+      port                  = "9083"
+      prefix                = "local1"
+      mapped-databases      = "default,test"
+      database-name-mapping = "test:test_alias,default:default_alias"
+      writable-whitelist    = "test"
+    }
+]
+``` 
+`local_metastores` map entry fields:
+
+Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| host | Host name of the Hive metastore server on the local network. | string | - | yes |
```

Comment:
There's a lot of repetition of this for local, remote and ssh. What do you think of breaking this up into a section for all metastores which covers the items which are common to all and then just having a section for each of the others which only has what's unique to them? What you have here now is actually pretty good for end users as they won't have to read two sections, but it's not so great for us to maintain. I'm also fine with this for now and changing later if we find we're updating this a lot (which we probably won't).